swagger.yaml

```diff
@@ -5193,6 +5203,18 @@ definitions:
       finalizing_stage_percentage:
         type: integer
 
+  disk-encryption:
+    type: object
+    properties:
+      enable_on:
+        type: string
+        description: Enable/disable disk encryption on master nodes, worker nodes, or all nodes.
+        enum: ['none', 'all', 'masters', 'workers']
+      mode:
+        type: string
+        description: The disk encryption mode to use.
+        enum: ['tpmv2', 'tang']
```

Comment:
For tang encryption, a user would need to specify a list of Tang URLs and Thumbprints.

Reply:
Seems like we will need additional info in the API as you first suggested after all @avishayt.

I didn't notice that the hard-coded URL is an example one.

Reply:
Good call.  For tang, we should add two parameters: URL and thumbprint.  See the ignition spec [here](https://coreos.github.io/ignition/configuration-v3_2/).  Note that this is unrelated to the "options" that you can also see in the spec, and which we removed for now:

> options (list of strings): any additional options to be passed to the cryptsetup utility.

Reply:
A list of {url, thumbprint} please :)

Reply:
Correct

Reply:
Fixed.